xslt/xml-to-html-statistics.xslt

```diff
@@ -32,15 +32,6 @@ Modified by Stephen Moon for DCO
 		xmlns:fn="http://www.w3.org/2005/xpath-functions"
 		exclude-result-prefixes="res xsl">
 
-  <xsl:template name="header">
-    <div>
-      <h2>Header</h2>
-      <xsl:for-each select="res:head/res:link"> 
-	<p>Link to <xsl:value-of select="@href"/></p>
-      </xsl:for-each>
-    </div>
-  </xsl:template>
-
```

Comment:
below here, line 58 or so, there's a <td> tag that needs to be removed along with its ending tag. We're not using tables in this xslt